packages/react-components/src/components/DevicesButton.tsx

```diff
@@ -303,7 +344,8 @@ export const DevicesButton = (props: DevicesButtonProps): JSX.Element => {
   const localeStrings = useLocale().strings.devicesButton;
   const strings = { ...localeStrings, ...props.strings };
 
-  const devicesButtonMenu = props.menuProps ?? generateDefaultMenuProps(props, strings);
+  const devicesButtonMenu =
```

Comment:
should generateDefaultDeviceMenuProps be memoized?

Reply:
No because,

(a) This wasn't memoized before my PR.
(b) `generateDefaultDeviceMenuProps` takes all `props` as input. No point memoizing because it'll be called on most renders anyway.